### PR TITLE
Feature/iteration 0 marceline

### PR DESCRIPTION
**Pull request by MarcelineBall**
Requesting code review on creation of random poster button

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 // query selector variables go here 👇
-
```
Comment (michannstoner) [inline src/main.js:2]: Excellent variable names!

File: src/main.js [hunk 2]
```diff
@@ -102,10 +105,15 @@ var savedPosters = [];
...
 // event listeners go here 👇
-
+randomPosterButton.addEventListener('click', makeRandomPoster)
 // functions and event handlers go here 👇
 // (we've provided one for you to get you started)!
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
-
+function makeRandomPoster() {
```
Comment (Meekb) [inline src/main.js:114]: Nice, clean code!

File: src/main.js [hunk 3]
```diff
@@ -102,10 +105,15 @@ var savedPosters = [];
...
+  mainPosterImage.src = images[getRandomIndex(images)]
+  posterTitle.innerText = titles[getRandomIndex(titles)]
+  posterQuote.innerText = quotes[getRandomIndex(quotes)]
+}
+makeRandomPoster()
```
Comment (Meekb) [inline src/main.js:119]: love makeRandomPoster function name! :)

File: src/main.js [hunk 4]
```diff
@@ -1,5 +1,8 @@
 // query selector variables go here 👇
-
+var randomPosterButton = document.querySelector('.show-random');
```
Comment (Meekb) [inline src/main.js:2]: var names are clear and concise

**Review by Meekb — APPROVED** [review]
Everything on features/iteration-0-Marceline branch looks great! Nice work.

**Review by michannstoner — APPROVED** [review]
This is great! Thank you for your hard work on iteration 0.

**Pull request merged**